### PR TITLE
Added a section about using Bootswatch themes

### PR DESCRIPTION
**Pull request by larsw**
...to the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,9 +241,27 @@ styleLoaders:
   - style
   - css?sourceMap
   - resolve-url?sourceMap
-  - sass?sourceMap
+  - sass?sourceMaph
```
Comment (alexfedoseev) [inline README.md]: typo?

Comment (larsw) [inline README.md]: Huh? this was probably added by the shitty keyboard on this Lenovo laptop.

**Comment by larsw** [comment]
Removed the typo in a new commit.

**Comment by justin808** [comment]
Thanks! This should help others, @larsw!

**Pull request merged**